### DISCUSSION TITLE
패킷에 광고가 기록되지 않는데 왜 그럴까요?

### DISCUSSION THREAD

**Discussion by ggoek34** [storycraft/node-kakao#765] [Q&A]
말 그대로 패킷에 광고가 기록되지 않으니 삭제도 안됩니다 ㅠㅠ

그냥 단순 링크는 아니고 노드카카오 이용해서 변형해서 보내는거 같은데 원인을 모르겠네요

특이사항은

오픈채팅방 링크를 걸었지만 오픈채팅방 미리보기 이미지가 뜨지 않습니다. 
카톡 피시버전으로 보면 답장하기에 메세지를 적은것으로 나오는데, 모바일버전으로 보면 그렇지 않구요. 

해결방법 아시는 분 있나요?

**Comment by punch5545** [comment]
#763 
on chat에 아예 잡히질 않네요

**Comment by holiday22** [comment]
카톡에서 패치를 하지 않는 이상 이미지 서칭말고는 가리기가 힘들겠네요.
